docs/contributing.md

```diff
@@ -1,70 +1,380 @@
+```bash
+git clone https://github.com/octocat/peopledepot.git
+```
+
+You should now have a new folder in your `hackforla` folder called `peopledepot`. Verify this by changing into the new directory:
+
+```bash
+cd peopledepot
+```
+
+#### **1.4.b Clone repo (2): Verify `origin` remote url**
+
+Verify that your local cloned repository is pointing to the correct `origin` URL (that is, the forked repo on your own Github account):
+```bash
+git remote -v
+```
+You should see `fetch` and `push` URLs with links to your forked repository under your account (i.e. `https://github.com/<your_GitHub_user_name>/peopledepot.git`). You are all set to make working changes to the website on your local machine.
+
+However, we still need a way to keep our local repo up to date with the deployed website. To do so, you must add an upstream remote to incorporate changes made while you are working on your local repo. Run the following to add an upstream remote URL & update your local repo with recent changes to the `hackforla` version:
 
-1. [Temporary] The latest code is in https://github.com/fyliu/peopledepot.git and we need to get that.
+```bash
+git remote add upstream https://github.com/hackforla/peopledepot.git
+git fetch upstream
+```
 
-   1. Create a git remote `fang` to that repo
+After adding the upstream remote, you should now see it if you again run `git remote -v` :
+```bash
+origin  https://github.com/<your_GitHub_user_name>/peopledepot.git (fetch)
+origin  https://github.com/<your_GitHub_user_name>/peopledepot.git (push)
+upstream        https://github.com/hackforla/peopledepot.git (fetch)
+upstream        https://github.com/hackforla/peopledepot.git (push)
+```
 
-       ```bash
-       git remote add fang https://github.com/fyliu/peopledepot.git
-       ```
+### **1.5 Dev setup (5): Set up Docker**
+Install or make sure [docker][docker-install] and [docker-compose][docker-compose-install] are installed on your computer
+```bash
+    docker -v
+    docker-compose -v
+```
 
-   1. Fetch data from the remote
+The recommended installation method for your operating system can be found [here](https://docs.docker.com/install/). <strong><em>Feel free to reach out in the [Hack for LA Slack channel](https://hackforla.slack.com/messages/people-depot/) if you have trouble installing docker on your system</em></strong>
 
-       ```bash
-       git fetch fang
-       ```
+More on using Docker and the concepts of containerization:
 
-   1. Checkout the recurring_event-model branch to see the current progress. The main repo should be updated with the latest code soon.
+* [Get started with Docker](https://docs.docker.com/get-started/)
 
-       ```bash
-       git checkout fang/recurring_event-model
-       ```
+### **1.6 Dev setup (6): Build and run the project locally with the script**
 
-1. Create .env.dev from .env.dev-sample
+**IMPORTANT:** Please make sure the `Docker Desktop` application is **running on your computer** before you run the bash commands below.
```

Comment:
Good point! How does the user know if docker desktop is running successfully? Maybe by running a simple command like docker ls? Disregard this if it's already covered somewhere.